plugin/core/views.py

```diff
@@ -520,20 +520,20 @@ def format_diagnostic_for_panel(diagnostic: Diagnostic) -> str:
     return formatted
 
 
-def format_diagnostic_related_info(info: DiagnosticRelatedInformation, base_dir: Optional[str] = None) -> str:
+def _format_diagnostic_related_info(info: DiagnosticRelatedInformation, base_dir: Optional[str] = None) -> str:
     file_path = info.location.file_path
     if base_dir and file_path.startswith(base_dir):
         file_path = os.path.relpath(file_path, base_dir)
     row = info.location.range.start.row + 1
     col = info.location.range.start.col + 1
     encoded_filename = "{}:{}:{}".format(info.location.file_path, row, col)
     file_path = "{}:{}:{}".format(file_path, row, col)
-    return '<a href="{}">{}</a>: {}'.format(encoded_filename, text2html(file_path), text2html(info.message))
+    return '<a href="file://{}">{}</a>: {}'.format(encoded_filename, text2html(file_path), text2html(info.message))
```

Comment:
Should we maybe use `filename_to_uri` on the `file_path` first?
I'm not sure what your change will do for Windows (the slashes might be wrong)

Reply:
Looking at the rest of the changes, it looks like this is actually pretty custom handling and not an URI.
So maybe to make that clearer, use a custom protocol like `location://`

Reply:
Good suggestion, thanks.